lib/pbench/server/api/resources/upload_api.py

```diff
@@ -298,6 +304,32 @@ def put(self, filename: str):
 
         try:
             dataset.advance(States.UPLOADED)
+
+            # NOTE: Metadata.SAVED and Metadata.SEEN are on the `dashboard` key
+            # while Metadata.DELETION is on `server`. We could still combine
+            # these two as `Metadata.create(dataset=dataset, key="dashboard",
+            # value={"saved": False, "seen": False})` and save a few DB
+            # operations. However, using separate `setvalue` operations has the
+            # advantage of not requiring any assumptions about the organization
+            # of the attributes, which is more maintainable.
+            Metadata.setvalue(dataset=dataset, key=Metadata.SAVED, value=False)
+            Metadata.setvalue(dataset=dataset, key=Metadata.SEEN, value=False)
```

Comment:
Are you going to retain these calls, or are you going to make the `dashboard` name space "open" and make the Dashboard set these?  (Or both?  :wink:)

Reply:
I haven't completely decided yet, although your instigations are definitely pushing me towards an open namespace with no server assumptions. That's going to mean more legacy test gold file updates, sadly. Maybe that's the main thing holding me back!

Reply:
Yeah...I'm wondering if there aren't some steps we can take to incrementally wean ourselves off of the gold files, or, at least, to make it so that they aren't quite so sensitive to changes like these.